python/ray/worker.py

```diff
@@ -806,6 +807,7 @@ def init(
         namespace: Namespace to use
         runtime_env: The runtime environment to use
             for this job (see :ref:`runtime-environments` for details).
+        metadata (list): List of tuples representing request metadata.
```

Comment:
PR looks good overall. Can we make this a bit more specific, i.e. "List of tuples representing request metadata for Ray Client gRPC."

Reply:
Addressed!